App/Containers/Dashboard.js

```diff
@@ -1,17 +1,78 @@
 import React from 'react'
-import { Text, View, ScrollView } from 'react-native'
+import Moment from 'moment'
+import { View, ScrollView, AsyncStorage } from 'react-native'
+import DashboardStat from '../Components/DashboardStat'
+import { default as StorageKeys } from '../Config/StorageKeys'
+import { Actions as NavigationActions } from 'react-native-router-flux'
+import { default as DeviceInfo } from '../Services/DeviceInfo'
 
 import styles from './Styles/DashboardStyle'
 
 export default class Dashboard extends React.Component {
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      steps: '',
+      distance: '',
+      deviceBattery: ''
+    }
+
+    this.fetchDayOfSummaryData()
+    this.fetchDeviceData()
+  }
+
+  async fetchDeviceData () {
+    let deviceInfo = await DeviceInfo.fetchDeviceData()
+
+    if (deviceInfo !== {}) {
+      this.setState({
+        deviceBattery: deviceInfo[0].battery
+      })
+    }
+  }
+
+  async fetchDayOfSummaryData () {
+    let date = Moment().format('YYYY[-]MM[-]DD')
+    let accessToken = await AsyncStorage.getItem(StorageKeys.FITBIT_ACCESS_TOKEN)
+
+    fetch(`https://api.fitbit.com/1/user/-/activities/date/${date}.json`, {
```

Comment:
If you're just doing for the current day, I think `today.json` may also work, but maybe once we abstract API stuff it won't matter.